supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -339,6 +339,17 @@ The API object for a deployment can be either a Kubernetes-native `Deployment` o
 
 *See also*:
 
+[discrete]
+[[dhcp]]
+==== image:images/yes.png[yes] DHCP (noun)
+*Description*: The Dynamic Host Configuration Protocol (DHCP) provides an IP address and other configuration information, such as IP addresses of DNS servers and time servers, to clients. DHCP clients use broadcasts to contact a DHCP server. Therefore, a DHCP server or a DHCP relay agent must be in the same broadcast domain as the client.
```

Comment:
Do we want to mention DHCP twice in the sentence >>suggestion>> Therefore, a DHCP server or relay agent must be....